src/main/scala/au/com/agiledigital/toolform/plugin/ToolFormPlugin.scala

```diff
@@ -0,0 +1,15 @@
+package au.com.agiledigital.toolform.plugin
+
+import cats.implicits._
+import com.monovore.decline._
+
+/**
+  * Extension point to add new commands to toolform.
+  * Uses Java SPI - See [[java.util.ServiceLoader]] for details.
+  * Implement the trait and register the new implementation in
+  * META-INF/services/au.com.agiledigital.toolform.plugin.ToolFormPlugin
+  * on the runtime classpath.
+  */
+trait ToolFormPlugin {
+  def command: Opts[Unit]
```

Comment:
Lets make this `Opts[Either[ToolFormError, String]]`

Reply:
Good suggestion, thanks!  This + moving the results printing back out to the app made the plugin world a better place.